Source/Core/InputCommon/GCPadStatus.h

```diff
@@ -48,7 +40,7 @@ struct GCPadStatus
   u8 triggerRight;  // 0 <= triggerRight <= 255
   u8 analogA;       // 0 <= analogA      <= 255
   u8 analogB;       // 0 <= analogB      <= 255
-  s8 err;           // one of PAD_ERR_* number
+  bool isConnected{true};
```

Comment:
I made it true by default to match the current behavior. I get what you're saying, but to me it makes more sense that if you create a struct where the most common usage is figuring out what the state of buttons/sticks/triggers etc are, then by default it should allow that usage.